src/views/Farms/Farms.tsx

```diff
@@ -133,10 +133,15 @@ const Farms: React.FC = () => {
   // Connected users should see loading indicator until first userData has loaded
   const userDataReady = !account || (!!account && userDataLoaded)
 
-  const [stakedOnly, setStakedOnly] = useState(!isActive)
+  const [stakedOnlyPersisted, setStakedOnlyPersisted] = usePersistState(null, {
```

Comment:
Why is the default value `null`?

Reply:
Null indicates that there is no user interaction yet therefore the defaul behaviour will be showing staked only farms when user clicks finished. When user toggles it, that behaviour will be ignored and selection from persist store will be applied

Reply:
I don't see why this is better than just adding the default value. You said yourself

> default behaviour which showing staked only farms when user clicks finished will occur.

Reply:
I updated the implementation to make it clean, and now there have a default value in persistent store

Reply:
@memoyil Sorry dude I am just not understanding why we have a default value if the default value should be `true` or `false`. I think we want to keep it simple and make staked only as `false`. That is the default state the page should render in if a user first lands on the page without visiting before.

Additionally, the enum you created is confusing. If I understand Typescript's enums `DEFAULT` is 0, `TRUE` is 1, and `FALSE` is 2.

Reply:
Because there is a feature there to show staked only when finished farms selected. In order to keep this i added default option. If user wants to disable the feature they have to toggle it.

Reply:
Okay, I will take your word for it. Please update the code to use the user reducer as @RabbitDoge noted below. Also, can you refactor the enum to be a bit more clear.

Reply:
I added strings to enums to make it more descriptive and changed persist state to user reducer.